packages/semver/src/executors/version/index.e2e.spec.ts

```diff
@@ -467,6 +468,31 @@ $`)
 
   describe('option --track-deps', () => {
     const mockCreateProjectGraphAsync = createProjectGraphAsync as jest.MockedFunction<typeof createProjectGraphAsync>;
+    const mockGetSortedProjectNodes = getSortedProjectNodes as jest.MockedFunction<typeof getSortedProjectNodes>;
+
+    describe('utilizes the project graph', () => {
+      beforeEach(() => {
+        const originalModule = jest.requireActual('@nrwl/workspace/src/core/project-graph');
+        mockCreateProjectGraphAsync.mockImplementation(originalModule.createProjectGraphAsync);
+        mockGetSortedProjectNodes.mockImplementation(originalModule.getSortedProjectNodes);
+      })
+      afterEach(() => jest.resetAllMocks());
+
+      it('uses a valid project graph version', async () => {
+        let errored = false;
+        try {
+          await getProjectDependencies('semver');
+        } catch (e) {
+          errored = true;
+        }
+        expect(errored).toEqual(false);
```

Comment:
What about this way?

```suggestion
        await expect(getProjectDependencies('semver')).rejects.not.toThrow();
```